APISpecification.yaml

```diff
@@ -475,3 +532,84 @@ components:
         - related_genomes_count
       additionalProperties: false
       type: object
+    Karyotype:
+      type: array
+      items:
+        $ref: '#/components/schemas/TopLevelRegion'
+    TopLevelRegion:
+      type: object
+      properties:
+        name:
+          type: string
+          example: "1"
+        type:
+          type: string
+          example: "chromosome"
+        length:
+          type: integer
+          example: 248956422
+        is_circular:
```

Comment:
Steve says there is no information about region topology in the metadata database. But hopefully, this is temporary. You can set this field to false until we start getting the correct data.